lib/transactions/vote.js

```diff
@@ -17,6 +17,14 @@ function createVote(secret, delegates, secondSecret) {
 		}
 	};
 
+	if (vendorField) {
+		transaction.vendorField = vendorField;
+
+    if(transaction.vendorField.length > 64){
```

Comment:
Since proposal is to make it 256 bytes can it get some kind of current global constant of bytes on how large is vendorfield??